tests/svg_path_test.py

```diff
@@ -78,3 +78,10 @@ def test_roundtrip_path_with_pen(d):
     pen = SVGPathPen()
     draw_svg_path(path, pen)
     assert pen.path.d == d
+
+
+def test_draw_svg_close_subpaths():
+    path = SVGPath(d="M0,0 L0,10 L10,10 L10,0 M12,0 L12,10 L22,10 L22,0")
+    pen = SVGPathPen()
+    draw_svg_path(path, pen, close_subpaths=True)
```

Comment:
Do we ever use close subpaths false? - if not perhaps we don't need a parameter

Reply:
well, only in the tests. I think I've used the `draw_svg_path` function while fiddling with SVG paths interactively in a font editor, it's still nice to be have to draw the _actual_ svg path with the fonttools pen, even if it's open.

Reply:
another reason to keep is being able to roundtrip draw_svg_path -> SVGPathPen  for self-testing purposes